.fvm/fvm_config.json

```diff
@@ -1,4 +1,4 @@
 {
-  "flutterSdkVersion": "3.18.0-0.2.pre",
+  "flutterSdkVersion": "3.18.0-0.2.pre@beta",
```

Comment:
flutter-fvm-config-action requires `@<channel>` annotation to determine which release channel to use: https://github.com/kuhnroyal/flutter-fvm-config-action/blob/34c3905bc939a4ff9d9cb07d5a977493fa73b2aa/index.js#L50-L52